test/parallel/test-exec-max-buffer.js

```diff
@@ -9,3 +9,14 @@ exec(cmd, { maxBuffer: 5 }, function(err, stdout, stderr) {
   assert.ok(err);
   assert.ok(/maxBuffer/.test(err.message));
 });
+
+if (!common.isWindows) {
```

Comment:
Is this because `echo` doesn't exist on windows? Is there anything we can do to test windows?

I'm not terribly comfortable with this kind of testing. :s